### ISSUE TITLE
document.changeTrackingMode does not work on Word Windows. document.load not registered

### ISSUE THREAD

**Issue by JensMadsen** [OfficeDev/office-js#2797]
When I use document.changeTrackingMode I get the "PropertyNotLoaded" error. This error is only seen in the windows Desktop app and not reproduced in Word Online. I have a gist which reproduces the error here: https://gist.github.com/JensMadsen/39bc5a9332941468ef3ebbb92a538a31

This is seen in Scriptlab as well. This problem influences our customers. Reason being that some of the range intersection logic in the API crashes when track changes are enabled, and hence we want to detect this in order to disable certain functionality in the add-in. I will provide another issue when I get to the bottom of the range intersection problems (later today).

## Your Environment

* Platform [PC desktop, Mac, iOS, Office on the web]: Pc, Windows 10
* Host [Excel, Word, PowerPoint, etc.]: Word
* Office version number: 1908,2108
* Operating System: Windows
* Browser (if using Office on the web): ______

## Steps to reproduce

The bug is only seen in windows Word desktop app. 

To reproduce run this in Word Windows (e.g. Scriptlab): 

```typescript 
async function run() {
  await Word.run(async (context) => {
    const document = context.document

    document.load("changeTrackingMode")   

    await context.sync();

    console.log(document.changeTrackingMode)
  });
}
```
or run this gist https://gist.github.com/JensMadsen/39bc5a9332941468ef3ebbb92a538a31 and you get the error 
![image](https://user-images.githubusercontent.com/6726111/186693492-f60e4b04-e77f-47ac-9dd2-3abb18d9baba.png)

I get the same error when I run the sample in the official documentation (https://docs.microsoft.com/en-us/javascript/api/word/word.document?view=word-js-preview#word-word-document-changetrackingmode-member): 
https://raw.githubusercontent.com/OfficeDev/office-js-snippets/prod/samples/word/50-document/manage-change-tracking.yaml

**Comment by JinghuiMS** [comment]
@JensMadsen Thanks for reaching us.
Regarding tracking change related APIs had been released on Word online, we will support it on Word Rich Client in the upcoming release. Meanwhile, you can try to update your office version to latest, you may access to these new APIs in advance.

**Comment by JensMadsen** [comment]
@JinghuiMS Thank you for the swift response. I did not know about online-only requirement sets. I read 1.1. Sorry. Our addin crashes when track changes is enabled due to range intersection operations (I think). Is there any other way to detect that track changes is enabled in Word Windows Desktop?

**Comment by JensMadsen** [comment]
@JinghuiMS a minimal demonstration of the problems we face are described here: https://github.com/OfficeDev/office-js/issues/2804

**Comment by JensMadsen** [comment]
@JinghuiMS If you can clarify if it is possible for an add-in in the Windows Word Desktop application to know if track changes is enabled or not I think we can close this issue?

**Comment by JinghuiMS** [comment]
@JensMadsen 
Currently, we may not have a way to detect that track changes are enabled on Word Desktop before the tracking change related APIs release. Why don't you try to update？

**Comment by JensMadsen** [comment]
@JinghuiMS I will try to update to the most recent version on another VM. But the problem is that our customers report and experience this problem. We cannot ask them to update so I need to find a more general solution I think. I have tried to extract the OOXML with and without Track changes so see if I am able to deduce it from there, but so far I do not see any solution going down that path.

**Comment by JinghuiMS** [comment]
@JensMadsen 
I can understand. Indeed, we tried to use getOOxml API to see if we can get something from it after you asked further question last week. Sorry we didn't find a way to unblock you.
